README.md

```diff
@@ -253,21 +297,20 @@ INFO: Checking pre-reqs for running TPC-DS queries is successful.
 INFO: Running TPCDS queries. Will take a few minutes depending upon the number of queries specified.. 
 INFO: Progress : [########################################] 100%
 INFO: TPCDS queries ran successfully. Below are the result details
-INFO: Individual result files: <TPCDS_JOURNEY_ROOT>/work/query<number>.res
-INFO: Summary file: <TPCDS_JOURNEY_ROOT>/work/run_summary.txt
+INFO: Individual result files: spark-tpc-ds-performance-test/work/query<number>.res
+INFO: Summary file: spark-tpc-ds-performance-test/work/run_summary.txt
 Press any key to continue
 ```
 
-### Option (7) - Cleanup toolkit
-This will cleanup all the files generated during setup steps 1, 2, 3, 4. If you use this option, make sure to run the setup steps ( 1, 2, 3, 4) before running queries using option (5, 6)
+#### Cleanup option: "(7) - Cleanup toolkit"
+
+This will clean up all of the files generated during option steps 1, 2, 3, and 4. If you use this option, make sure to run the setup steps ( 1, 2, 3, 4) before running queries using option 5 and 6.
 
-### Option (Q) - Quit
-This will exit the script
+#### Cleanup option: "(Q) - Quit"
 
-## Running using Notebook.
-### Steps
+This will exit the script.
 
-Follow these steps to setup and run this developer journey. These steps are described in detail below.
+## Run using a Jupyter notebook in the IBM Data Science Experience (DSX) on IBM Bluemix
```

Comment:
I'd shorten this a pinch to "Run using a Jupyter notebook in the IBM Data Science Experience"